Resources/Prototypes/Datasets/Names/borg.yml

```
@@ -1,6 +1,18 @@
+# SPDX-FileCopyrightText: 2025 MilenVolf <63782763+MilenVolf@users.noreply.github.com>
+# SPDX-FileCopyrightText: 2025 darkdan <145926356+Starbuckss14@users.noreply.github.com>
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
 # only used roundstart, names can be changed after
 - type: localizedDataset
   id: NamesBorg
   values:
     prefix: names-borg-dataset-
-    count: 45
+    count: 43
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)